### ISSUE TITLE
[Suggestion] Night Events

### ISSUE THREAD

**Issue by Borjoize** [parabola949/Werewolf#72]
Hey, was thinking about something to add some new dynamics to the game.

Night Events

Every X nights there is a night event. These events occur randomly and they can give advantage and disadvantages to any team/person. I guess about every 4 or 5 nights is a good number, and they would only work on games with 10+ players. Here are some suggestions  on possible events:

Full Moon - Werewolves get to kill 2 people this night, cuz, Full Moon is they're moon bro.

New Moon - Werewolves get no kill this night,  cuz is to dark? Maybe they are afraid of dark.

Blue Moon - Seer gets to see 2 people this night, cuz this is a magical night. Or, the Fool and/or the aprendice could see someone?

Red Moon - The Serial Killer kills twice, and the harlot visits to people in this sinfull night.

Eclipse - A Holy, or Unholy night, who knows? Cult can convers two people, and cultist hunter can visit two people too!

Snowstorm - Only wolves can act this night, cuz they're furry and can resist such cold!

Rainy Night - Only non-wolves can act this night! Cuz no one wants to wet a fur coat

Thunderstorm - Random person gets struck by lightining, and dies

Sandstorm - Every one powers will chose targets randomly, cuz no one can see clearly on the sandstorm.

Flood - No actions for anyone this night.

**Comment by BlaQPhoeniX** [comment]
@Borjoize That's really unbalanced

**Comment by limatiga** [comment]
I'd love to see this, if para accept it.
But I think every game need at least 20 players to activate the night event, also in /config it must be added "Allow night event" option.

**Comment by Borjoize** [comment]
@BlaQPhoeniX yeah, true, but is it bad? I mean is not an actual "on bios unbalance" for some side(as it is now) it is an event to give some randomness to the match. So it can unbalance the game for both sides, in that sense, it's balanced in the end.

@limatiga completely needs to be a /config thing, since it adds randomness and not balance, and not everyone wants that.

**Comment by ijat** [comment]
Cool stuff !

**Comment by Olfi01** [comment]
This adds another whole new dimension to the game, and I like the idea! Of course, it should be configurable.
I'll take a note to maybe do this
